src/scripts/visualizations/tooltip.js

```diff
@@ -97,10 +98,21 @@
                     };
 
                     return clampPosition(pos);
+                },
+
+                'cursor': function cursorPositioner() {
+                    var pos = d3.mouse(layer[0][0]);
+                    var tooltip = tooltipElement[0][0]
+                    var offsetX = 40;
```

Comment:
I see why you need to pick some distance though. Maybe you could just mark this as a constant (all caps, snake case is preferred notation in pre-es6 javascript **I think).